src/styleguide/analyzer/typescript-react-analyzer/property-analyzer.ts

```diff
@@ -214,7 +214,10 @@ export class PropertyAnalyzer {
 
 			if (objectType.objectFlags & ts.ObjectFlags.Interface) {
 				const property = new ObjectProperty(args.symbol.name);
-				property.setProperties(PropertyAnalyzer.analyze(args.type, args.typechecker));
+				// tslint:disable-next-line:arrow-return-shorthand
+				property.setPropertyResolver(() => {
```

Comment:
instead of disabling the linting rule you could just write this:
`property.setPropertyResolver(() => PropertyAnalyzer.analyze(args.type, args.typechecker))`

Reply:
oh yes sorry i forgot that line. i had a debug breakpoint there before :)
thx for finding it